spec/test-outputs/www-integration.out.vcl

```diff
@@ -510,5 +518,21 @@ sub vcl_pass {
 }
 
 sub vcl_hash {
+  # include request method
+  set req.hash += req.request;
+
+  # support caching non-GET/HEAD requests
+  if (!(req.request == "GET" || req.request == "HEAD" || req.request == "FASTLYPURGE") && req.postbody) {
+    set req.hash += req.postbody;
+  }
+
+  # if handling GET/HEAD requests, fall back to normal
```

Comment:
I don't understand this comment, doesn't the following code run regardless of request method?